.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
Peux-tu supprimer le dossier `.vs/` de la PR, et ajouter ça dans to gitignore global :

```gitignore
.vs/
```